csrankings.csv

```diff
@@ -7379,6 +7379,7 @@ Junmo Kim,KAIST,https://sites.google.com/site/siitkaist/professor,GdQtWNQAAAAJ
 Junping Sun,Nova Southeastern University,http://scis.nova.edu/~jps,NOSCHOLARPAGE
 Junsong Yuan,University at Buffalo,https://cse.buffalo.edu/~jsyuan,fJ7seq0AAAAJ
 Junya Honda,University of Tokyo,http://www.it.k.u-tokyo.ac.jp/~honda/index_e.html,NOSCHOLARPAGE
+Junyi Jessy Li,University of Texas at Austin,https://jjessyli.github.io/,tJGm3-YAAAAJ

```

Comment:
Remove. Not on this page: https://www.cs.utexas.edu/faculty, not eligible for inclusion.